fastlane/Fastfile

```diff
@@ -431,19 +431,28 @@ desc 'Runs tests in Debug config'
 lane :test do |options|
   next unless check_required?(:unit)
 
-  add_ci_env_var_to('../Tests/StreamChatTests/StreamChatTestPlan.xctestplan')
-
-  number_of_retries = options[:cron] ? 1 : 0
+  add_ci_env_var_to('../Tests/StreamChatTests/StreamChatFlakyTests.xctestplan')
 
   scan(
     project: 'StreamChat.xcodeproj',
     scheme: 'StreamChat',
-    testplan: 'StreamChatTestPlan',
-    configuration: 'Debug',
+    testplan: 'StreamChatFlakyTests',
     clean: true,
     devices: options[:device],
     build_for_testing: options[:build_for_testing],
-    number_of_retries: number_of_retries
+    number_of_retries: 2
```

Comment:
I think it is better we increase it to 5. 2 might be too little 🤔

Reply:
`2` means fastlane will try 3 times (`1 try` + `2 retries`). Let's set up `4` then?

Reply:
Bumped to `5` retries 👍